app/src/main/java/com/neosoft/neostore/ServiceAPI/GetServices.java

```diff
@@ -0,0 +1,44 @@
+package com.neosoft.neostore.ServiceAPI;
+
+import android.util.Log;
+
+import com.neosoft.neostore.Activity.LoginActivity;
+
+import okhttp3.FormBody;
+import okhttp3.RequestBody;
+
+import static com.neosoft.neostore.ServiceAPI.UserAPI.LOGIN_URL;
+import static com.neosoft.neostore.ServiceAPI.UserAPI.REGISTER_URL;
+
+public class GetServices implements Services.ApiResponse{
+
+    public void login(String email, String pass)
+    {
+        RequestBody requestBody = new FormBody.Builder()
+                .add("email",email)
+                .add("password",pass)
+                .build();
+        Services ser = new Services(LOGIN_URL, requestBody, this);
+        ser.execute(requestBody);
+    }
+
+    public void register(String fname, String lname, String email, String pass, String cpass, String gender, String ph_no)
+    {
+        RequestBody requestBody = new FormBody.Builder()
+                .add("first_name",fname)
+                .add("last_name",lname)
+                .add("email",email)
+                .add("password",pass)
+                .add("confirm_password",cpass)
+                .add("gender",gender)
+                .add("phone_no",ph_no)
+                .build();
+        Services ser = new Services(REGISTER_URL, requestBody, this);
+        ser.execute(requestBody);
+    }
+
+    @Override
+    public void onSuccess(String response) {
```

Comment:
I don't think GetServices needs any response. Rather Activity should have onSuccess listener.